beacon/sync/src/integration-test/java/tech/pegasys/teku/beacon/sync/BlockPropagationIntegrationTest.java

```diff
@@ -69,11 +69,12 @@ public void shouldFetchUnknownAncestorsOfPropagatedBlock() throws Exception {
 
     // Setup node 2
     SyncingNodeManager node2 =
-        SyncingNodeManager.create(
-            asyncRunner,
-            networkFactory,
-            validatorKeys,
-            c -> c.rpcEncoding(rpcEncoding).gossipEncoding(gossipEncoding));
+        SyncingNodeManager
+            .create( // FIXME minimal phase0 so going to get empty result for blocks (line 99)
```

Comment:
We're not adding a fixme to master... what's the plan here? is this just a note that we need to clean up or is there still work to be done?